src/runtime/importhook.cs

```diff
@@ -43,7 +43,6 @@ internal static void Initialize()
             py_import = Runtime.PyObject_GetAttrString(mod, "__import__");
             hook = new MethodWrapper(typeof(ImportHook), "__import__", "TernaryFunc");
             Runtime.PyObject_SetAttrString(mod, "__import__", hook.ptr);
-            Runtime.XDecref(hook.ptr);
```

Comment:
I put in some of these changes in #993. 

That patch does not include the change to how you release the hook, and does not include the CLRModule.Reset()